docker-compose.yml

```diff
@@ -9,17 +8,23 @@ services:
         volumes:
             - postgres12:/var/lib/postgresql/data
         restart: on-failure
+
+        # If we want to connect to the server from outside
+        ports:
+          - 5432:5432
+
     mauro-data-mapper:
         image: "maurodatamapper/mauro-data-mapper:${MDM_TAG}"
         build:
             context: mauro-data-mapper
             args:
                 MDM_APPLICATION_COMMIT: "${MDM_APPLICATION_COMMIT}"
                 MDM_UI_COMMIT: "${MDM_UI_COMMIT}"
-                ADDITIONAL_PLUGINS: ""
-                MDM_UI_THEME_NAME: "default"
+                ADDITIONAL_PLUGINS: "uk.ac.ox.softeng.maurodatamapper.plugins:mdm-plugin-nhs-data-dictionary:2.0.0-SNAPSHOT"
```

Comment:
Good start, and I've tested the NHS plugin endpoints in my container which all seem to work. We might need to add additional plugins later, but I'm not sure what those will be yet - this is fine for now.